docs/source/evaluation.rst

```diff
@@ -71,20 +71,20 @@ How-To
 
 Given a serialized state of a fully optimized trajectory (here: ``optimized.pbstream`` file), the ground truth relations can be generated with the following command:
 
-.. code-block::
+.. code-block:: console
   
   cd <build>  # (directory where Cartographer's binaries are located)
   ./cartographer_autogenerate_ground_truth -pose_graph_filename optimized.pbstream -output_filename relations.pbstream -min_covered_distance 100 -outlier_threshold_meters 0.15 -outlier_threshold_radians 0.02
 
 Then, a non-optimized trajectory ``test.pbstream`` can be evaluated against the generated relations with:
 
-.. code-block::
+.. code-block:: console
```

Comment:
Consistency with other Cartographer documentation: `bash`? Similar above.

Reply:
I had `bash` initially, but since we also have plain text and not only bash it looks odd in some details, see the zeros in the attached image.
![screenshot from 2018-02-02 14-49-15](https://user-images.githubusercontent.com/8985495/35736426-51633da8-0828-11e8-9314-5d1b4b344653.png)

I can also change the commands to `bash` and leave the output as `console` - probably the best solution?